### ISSUE TITLE
TV Device (ATV/Fire Stick/etc) UI + DPAD Support

### ISSUE THREAD

**Issue by JLDevUsr** [ApolloTVofficial/kamino#111]
It would be cool to have an AndroidTV Layout compatible with dpad support to enable AndroidTV users to navigate and do so smoothly.

**Comment by kevincosmos** [comment]
Hi @JLDevUsr, TV UI is already in development, we don't have an ETA for it currently. This will incorporate DPAD support. Some mockups can be found here: https://imgur.com/a/7eooPmz.

**Comment by kevincosmos** [comment]
@NBTX BradB has requested Android TV "Channels" integration, I believe TVZion has this implemented if you were wanting to see how it works.

**Comment by NBTX** [comment]
This issue depends on the status of https://github.com/flutter/flutter/pull/30040 as we cannot implement focus-based navigation until the PR has landed into the Flutter code base (`beta` or ideally `stable` branch).

**Comment by kevincosmos** [comment]
Removed pin as this is waiting on framework update rather than us

**Comment by NBTX** [comment]
Closing in favor of tracking the framework updates.
